### PR TITLE
 ✨ New ad type: WPMedia 

### PR DESCRIPTION
**Pull request by LucasRosaWP**
Hi,
This is the first version of the component for testing purposes.
We need it to test ads on our first AMP site 'sportowefakty.wp.pl' (in development).

Regards
Łukasz Rosa

### CODE REVIEW

**Comment by LucasRosaWP** [comment]
I signed the CLA!

**Comment by LucasRosaWP** [comment]
OK. Authorized signer Paweł Karaszewski (Pawel.Karaszewski@grupawp.pl) signed the CLA.

**Comment by LucasRosaWP** [comment]
Good day,
I have a problem with the CLA required for the AMP project.
My supervisor (VP IT) signed the CLA as an authorized signer.
We want to withdraw this CLA and sign a new one, in which I will be an authorized signer because now we can not contribute to the APM project.
How can I do this?

Regards
Łukasz Rosa
lukasz.rosa@grupawp.pl

**Comment by LucasRosaWP** [comment]
OK, What is the next step?

**Comment by lannka** [comment]
@LucasRosaWP could you please follow the guideline [here](https://github.com/ampproject/amphtml/blob/master/ads/README.md#developer-guidelines-for-a-pull-request) and implement render-start signal?

**Comment by LucasRosaWP** [comment]
@lannka OK, just implemented render-start signal.

File: ads/_config.js [hunk 1]
```diff
@@ -879,6 +879,10 @@ export const adConfig = {
 
   widespace: {},
 
+  wpmedia: {
+    prefetch: 'https://std.wpcdn.pl/wpjslib/wpjslib-amp.js',
+  },
```
Comment (lannka) [inline ads/_config.js]: you will need to add
```
renderStartImplemented: true,
```
to opt in.

**Comment by LucasRosaWP** [comment]
@lannka I don't know why, but forgot to push my all chages. Done.

**Comment by LucasRosaWP** [comment]
@lannka Thank you :)
When can we expect a release containing our changes?

**Pull request merged**